### PR TITLE
add r-geofacet, r-geogrid, r-imgur

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@ndmaxar @RyanHackney agree to be maintainers?

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/r-geofacet/build.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (dbast) [inline recipes/r-geofacet/build.sh:8]: Please remove the mv and grep line.

File: recipes/r-geogrid/build.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (dbast) [inline recipes/r-geogrid/build.sh:8]: Please remove the mv and grep line.

File: recipes/r-imgur/build.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (dbast) [inline recipes/r-imgur/build.sh:8]: Please remove the mv and grep line.

File: recipes/r-imgur/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
```
Comment (dbast) [inline recipes/r-imgur/meta.yaml:25]:
```suggestion
    - {{ posix }}zip               # [win]
```

File: recipes/r-geogrid/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
```
Comment (dbast) [inline recipes/r-geogrid/meta.yaml:29]:
```suggestion
    - {{ posix }}filesystem        # [win]
```
Same for the other lines.

File: recipes/r-geofacet/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
```
Comment (dbast) [inline recipes/r-geofacet/meta.yaml:25]:
```suggestion
    - {{ posix }}zip               # [win]
```

File: recipes/r-geofacet/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/geofacet_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/geofacet/geofacet_{{ version }}.tar.gz
+  sha256: 5c908e4b046ea0c1ff5c4ecd144f3b8773598b06ee87afc6f1e54ccca1e8d4ab
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (dbast) [inline recipes/r-geofacet/meta.yaml:18]: Please add:
```suggestion
  number: 0
  noarch: generic
```

File: recipes/r-geogrid/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/geogrid_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/geogrid/geogrid_{{ version }}.tar.gz
+  sha256: 07c54dac348200b5dc4c610fce643d672f1e273ebe8802bff9a6f4241f770a2d
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (dbast) [inline recipes/r-geogrid/meta.yaml:18]: Please add:
```suggestion
  number: 0
  noarch: generic
```

File: recipes/r-imgur/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/imguR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/imguR/imguR_{{ version }}.tar.gz
+  sha256: 6311f971d9a2041250e45c81184d0408f220f37236e62016be7965c41e7cc791
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (dbast) [inline recipes/r-imgur/meta.yaml:18]: Please add:
```suggestion
  number: 0
  noarch: generic
```

Comment (rluria14) [reply recipes/r-imgur/meta.yaml:18]: @dbast seems that noarch: generic wouldn't work here

**Comment by dbast** [comment]
@rluria14 Thanks!

As the package depend on each other, this once requires carefully re-triggering CI in right order after feedstock conversions to upload the packages. Will take care of that.

**Comment by dbast** [comment]
All three packages are now available.

**Pull request merged**